operator/src/simulation/manager.rs

```diff
@@ -51,8 +51,8 @@ pub fn manager_job_spec(config: ManagerConfig) -> JobSpec {
                 subdomain: Some("goose".to_owned()),
                 containers: vec![Container {
                     name: "manager".to_owned(),
-                    image: Some("keramik/runner:dev".to_owned()),
-                    image_pull_policy: Some("IfNotPresent".to_owned()),
+                    image: Some("public.ecr.aws/r5b3e0r5/3box/keramik-runner:latest".to_owned()),
+                    image_pull_policy: Some("Always".to_owned()),
```

Comment:
Did we want the `Always` here?

Reply:
yes, for the `latest` tag we want `Always` or we will not ever see new updates.